src/pages/_app.tsx

```diff
@@ -15,10 +14,8 @@ const MyApp: FC<AppProps> = ({ Component, pageProps }) => {
     const rdxStore = store();
     return (
         <Provider store={rdxStore}>
-            <IsUserAuthorizedContext>
-                <ToastContainer />
-                <Component {...pageProps} />
-            </IsUserAuthorizedContext>
+            <ToastContainer />
+            <Component {...pageProps} />
```

Comment:
what this file is doing??

Reply:
ToastContainer might have been using the value provided by the IsAuthorizedContext . Since I removed the context , there was no need for it to be there